main.py

```diff
@@ -105,10 +105,11 @@ def parse(soup):
     while True:
         try:
             print(input(f'Paste the whole page source at {HTML_FN}, and press Enter when you\'re done:\n'))
-            html_doc = ''.join(map(bytes.decode, open(HTML_FN, 'rb').readlines()))
+            with open(HTML_FN, 'r') as f:
+                html_doc = ''.join(f.readlines())

```

Comment:
That also means we still have to use `bytes.decode`

Reply:
Thank you for your valuable and insightful comments! Definitely agree with your suggested changes! Fixed in [0872f78](https://github.com/RussellDash332/pancake/pull/1/commits/0872f7833bfbd33fa5595da0ff2c0b0268698518).